plugins/plugin-git/che-plugin-git-ext-git/src/main/resources/org/eclipse/che/ide/ext/git/client/panel/git.svg

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!--
+
```

Comment:
This SVG file is an icon with just `git` word. I think it looks redundant in the tab near the `Git` name (git Git)

Reply:
Well, for CHE 5 it is probably redundant, but not critical.
As you probably know, CHE 6 has only images on tabs without any text title, and there it will look quite good.

Reply:
It could be easily changed, I just followed CHE 6 mockup.

Reply:
We can have a git logo icon that can fit both CHE-5 and CHE-6.

Reply:
Ok, I'll add git logo image in the next issue related to this panel.